private-templates-service/client/src/App.tsx

```diff
@@ -110,6 +113,10 @@ class App extends Component<Props, State> {
 
     return (
       <Router>
+        <IdleTimer
+          element={document}
+          onIdle={this.onIdle}
+          timeout={1800000} />
```

Comment:
keep it in a separate constants file/collection?

Reply:
Done :)